### ISSUE TITLE
Error message when running npm run dev

### ISSUE THREAD

**Issue by vegarrorvik** [sondretjvimond/UiBFlix#1]
Me and a couple of other students are getting this error message when running "npm run dev":

ERROR in ./lib/ajax/request.js
Module not found: Error: Cannot resolve module 'xhr' in /Users/ady/WebstormProjects/UiBFlix/project/lib/ajax
 @ ./lib/ajax/request.js 7:11-25

The localhost:8080 is blank - is it supposed to be?

**Comment by sondretjvimond** [comment]
Forgot to add dependencies to xhr it seems. Should work if you run npm install xhr.

**Comment by sondretjvimond** [comment]
You should see a videoplayer when visiting localhost:8080.

**Comment by vegarrorvik** [comment]
Yeah that worked well and it seems to be running smoothly. 
 
I have some unmet dependencies and some other things flashing red - I can send some of it to you?

Vegar

**Comment by sondretjvimond** [comment]
Yes, sure!

**Comment by vegarrorvik** [comment]
These were the once I picked up, maybe lost some because of the terminal not showing everything.

http://pastebin.com/L8ZAKTd3 <http://pastebin.com/L8ZAKTd3>

Vegar

**Comment by sondretjvimond** [comment]
Yeah, this happens on other operating systems than MacOS. I'll see if I can remove it, it wont affect the project though. Let me know if there is anything more.

**Comment by vegarrorvik** [comment]
Okay, running OSX, not currently updated to MacOS. Happens on windows as well as you already pointed out. Nevertheless it runs on my machine, two other mac's and a windows machine, so I think we’re all set.
